client/rpc/generate-api.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Generates Javascript stubs for our API
+
+cd $(dirname "$0")/..
+
+# TODO: Update output directory
+java -jar lib/present-rpc-compiler.jar --proto_path=../proto --js_out=react/src
```

Comment:
What's the versioning, license, etc. story with this JAR?  Do we need the binary checked in directly?

Reply:
This is my project. It's Apache: https://github.com/presentco/present-rpc

I went with checking in a jar to simplify dev environment config. The jar is really small.

Reply:
Since we're copying the binary in to the repo can we 1) add a comment here of exactly how to reproduce it (I assume just https://github.com/presentco/present-rpc and a particular git sha), and 2) add to THIRD_PARTY_LICENSE your license details.

Reply:
Linked to download instructions and updated license file.

BTW, this tool just generates Javascript functions from proto rpcs. Easy enough to code these by hand, too.